airbyte-workers/src/main/java/io/airbyte/workers/DefaultNormalizationWorker.java

```diff
@@ -64,11 +65,16 @@ public Void run(final NormalizationInput input, final Path jobRoot) throws Worke
       LOGGER.info("Normalization was cancelled.");
     }
 
-    final Duration duration = Duration.ofMillis(System.currentTimeMillis() - startTime);
+    final long endTime = System.currentTimeMillis();
+    final Duration duration = Duration.ofMillis(endTime - startTime);
     final String durationDescription = DurationFormatUtils.formatDurationWords(duration.toMillis(), true, true);
     LOGGER.info("Normalization executed in {}.", durationDescription);
 
-    return null;
+    final StandardNormalizationSummary summary = new StandardNormalizationSummary()
+        .withStartTime(startTime)
+        .withEndTime(endTime);
+
+    return summary;
```

Comment:
Could you update the `DefaultNormalizationWorkerTest` to verify the output of this method?